src/sentry/utils/json.py

```diff
@@ -63,6 +63,11 @@ def better_default_encoder(o):
 )
 
 
+def dump(value, fp, **kwargs):
```

Comment:
Yeah, I did realize it was probably out of the scope of this review. I still don't think it's a good pattern, though.